python/ctsm/toolchain/ctsm_case.py

```diff
@@ -266,122 +265,122 @@ def create_namelist_file(self):
             self.create_landuse_file()
 
         self.build_namelist_filename()
-        namelist_file = open(self.namelist_fname, "w")
+        with open(self.namelist_fname, "w", encoding='utf-8') as namelist_file:
```

Comment:
@slevisconsulting I just wanted to say thanks for the update here from `open` to `with open`. I read about it a bit [online](https://stackoverflow.com/questions/34429519/python-difference-between-open-and-with-open) and it seems like using `with open` is the safer way to write a file compared to using `open`. I learnt this from your update, so just wanted to say thanks! 👍

Reply:
I found this from trying to eliminate pylint warnings :-)